src/main/java/io/seqera/tower/cli/commands/datasets/CreateCmd.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.datasets;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.commands.global.WorkspaceOptionalOptions;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.cli.responses.datasets.DatasetCreate;
+import io.seqera.tower.model.CreateDatasetRequest;
+import io.seqera.tower.model.CreateDatasetResponse;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+@CommandLine.Command(
+        name = "create",
```

Comment:
Rename the command and the class to `add` as we did with all the other `create` subcommands that we had.